### PR TITLE
Zed kolla build config

### PR DESCRIPTION
**Pull request by Alex-Welsh**
After this PR, Zed kolla container images should all be buildable and AIO CI should pass (the branch protection rules do still need a bit of work though)

### CODE REVIEW

**Comment by Alex-Welsh** [comment]
AIOs should pass, but it's pulling the wrong version of kayobe-automation https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6014619558/job/16314730497?pr=606#step:2:482 I'll see if I can get it fixed tomorrow morning

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -6,63 +6,14 @@
...
-octavia_tag: yoga-20230523T110936
-openvswitch_tag: yoga-20230515T150233
-ovn_tag: yoga-20230515T150233
-
 # kolla_base_distro must be set here to be resolvable on a per-host basis
 # This is necessary for os migrations where mixed clouds might be deployed
-kolla_base_distro: "{% raw %}{{ 'centos' if ansible_facts.distribution == 'Rocky' and ansible_facts.distribution_major_version == '8' else ansible_facts.distribution | lower }}{% endraw %}"
+kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
 
 kayobe_image_tags:
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml:13]: I don't see this referenced anywhere.

Comment (Alex-Welsh) [reply etc/kayobe/kolla/globals.yml:13]: It will be necessary again as soon as we update an individual tag, but probably not needed now

Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml:13]: I thought we were going to use `openstack_tag` in globals.yml to set the default tag?

**Review by markgoddard — APPROVED** [review]

**Pull request merged**